suites/pacific/cephfs/tier_1_fs.yaml

```diff
@@ -0,0 +1,126 @@
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+          - config:
+              command: shell
+              args:          # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply
+              service: mds
+              base_cmd_args:          # arguments to ceph orch
+                verbose: true
+              pos_args:
+                - cephfs              # name of the filesystem
+              args:
+                placement:
+                  label: mds
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+        abort-on-fail: true
+        config:
+            command: add
+            id: client.1
+            node: node2
+            install_packages:
+                - ceph-common
+            copy_admin_keyring: true
+        desc: Configure the Cephfs client system 1
+        destroy-cluster: false
+        module: test_client.py
+        name: configure client
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.2
+        node: node3
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Configure the Cephfs client system 2
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.3
+        node: node1
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Configure the Cephfs client system 3
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.4
+        node: node5
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Configure the Cephfs client system 4
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+  - test:
+        name: cephfs-volume-management
+        module: cephfs_volume_management.py
```

Comment:
Instead of creating a new module, could you check if we can reuse `exec.py` module

Reply:
I want to enhance more to this , so I thought of having a separate module.